projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.component.html

```diff
@@ -171,3 +171,7 @@
         </igx-excel-style-filter-operations>
     </igx-grid-excel-style-filtering>
 </div>
+
+<div #sink style="display: none;">
```

Comment:
@MayaKirova Minor leftover from the old comment about the pivot was that "one of the outlets is marked to keep the query working" so there are now two `#sink` items in the template. Should be fine to remove it from the outlet container.